Article/Article.js

```diff
@@ -26,4 +28,8 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");
+
+articles.forEach( (domElement) => {
+  return new Article(domElement);
+})
```

Comment:
The `.forEach` method runs like a regular for loop. Since it doesn't return a new array or anything like that, you can simply call `new Article()` without the `return` keyword beforehand.